demo/play-demo-for-recording.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+#asciinema rec -w 2
+
+exec() {
+  echo "\$ $@"|pv -qL 10 ; "$@" ;
+  }
+
+echo "   "
+echo "   "
+echo " Simplify the deployment of Spring Boot applications using Halkyon Component Operator on Kubernetes"|pv -qL 10
+echo " In this demo, we will :"|pv -qL 10
+echo "   --> Compose & link 2 microservices: client and fruits backend"|pv -qL 10
+echo "   --> Deploy a capability such as a database and link it to the backend microservice accessing it"|pv -qL 10
+echo "   --> Code locally and next push/build on Kubernetes/OpenShift"|pv -qL 10
+echo "  "|pv -qL 10
+echo " Ready? Let's begin!"|pv -qL 10
+echo "  "
+echo "  "
+sleep 5
+
+clear && sleep 1
+echo "# Log on to the cluster using the oc client"|pv -qL 10
+sleep 2
+exec oc login https://159.69.209.188:8443 --token=5tKSUvACZ8big9XT2mONlqMU5J4w6Di6RFe9wrnJiU0
```

Comment:
I don't think it's a good idea to show this step at all: we shouldn't advertize our cluster IP or its token (even though it is expired).

Reply:
Yes, I agree, I was wondering about it also and finally didn't share the idea.